consul/src/main/java/org/secondbase/consul/ConsulModuleConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package org.secondbase.consul;
+
+import org.secondbase.flags.Flag;
+
+/**
+ * Configuration parameters for the WebConsole.
+ */
+public final class ConsulModuleConfiguration {
```

Comment:
Maybe we can come up with a rule we only have this separate class if there's more than five parameters? We could save quite some space by reintegrating this in the module classes everywhere. But we can do this later.